client/elements/text/sc-simple-text.html

```diff
@@ -781,6 +830,51 @@ <h3>Could not find the sutta text.</h3>
                   }, 0);
               }
           }
+
+          _markupVarNote(noteData,referenceText) {
+              const noteItems = noteData.split(/ \| /);
+              var noteText = referenceText + '<paper-tooltip class="varnote"><table><tbody>';
+              var out = "";
+
+              for (var item in noteItems) {
+                var m = noteItems[item].match(/^(\(.*?\)|[^\(]+)(.*)/),
+                  variant = m[1],
+                  comment = m[2];
+                if (variant && !comment) {
+                    // In some cases there is no variant, it's simply a comment
+                    // In this case, the comment takes the entire line.
+                    out = '<tr><td colspan=2>' + this._markupComment(variant) + '</td></tr>';
+                } else {
+                    // The normal case is a variant and a comment or qualification
+                    // pertaining to that variant reading.
+                    out = '<tr><td>' + variant + '</td><td>' + this._markupComment(comment) + '</td></tr>';
+                }
+                noteText += out;
+              }
+              return noteText + '</tbody></table></paper-tooltip>';
+          }
+
+          _markupComment(comment){
+              comment = comment.replace('(','').replace(')','');
```

Comment:
Please use whitespace after `,` for easier reading
`comment.replace('(', '').replace(')', '');`